docs/backup_and_restore.rst

```diff
@@ -104,6 +104,30 @@ archive in the output of the backup command.
 
 .. include:: includes/backup-warning.txt
 
+.. _backing_up_logos:
+
+Backing up your custom logo
+'''''''''''''''''''''''''''
+
+As of SecureDrop |version|, the backup will not include any custom logo file
+that you may have configured for your SecureDrop instance. To back up the logo,
+download it using Tor Browser or on the command line. From ``~/Persistent``
+(a level above the ``securedrop`` directory), the following command will
```

Comment:
Suggested going to `~/Persistent` because this file is not included in `.gitignore`, so it would mess up future `git status` output. (https://github.com/freedomofpress/securedrop/issues/3502 can't come soon enough.)